example/views.py

```diff
@@ -1,8 +1,8 @@
-from django.shortcuts import render_to_response
+from django.shortcuts import render
 from .forms import DatesForm
 
-
```

Comment:
The removal of this line (and extra blank line at the end) cause the `flake8` test to fail.